### ISSUE TITLE
Creating new files using its admin page

### ISSUE THREAD

**Issue by pli888** [gigascience/gigadb-website#18]
There is a problem with using the create new file admin page. After filling in the required details on the create new file web page form and clicking the "Create" button, the nginx web server responds after about 1 minute with a 502 Bad Gateway error message. However, the file is created and associated with the correct dataset when you subsequently click on the dataset link.

The 502 Bad Gateway error needs to be fixed.

**Comment by only1chunts** [comment]
Related (possibly the same thing?) I get a CDbException:
"CDbCommand failed to execute the SQL statement: SQLSTATE[22007]: Invalid datetime format: 7 ERROR:".... followed by 2 pages of geek talk
Basically I think this error arises due to me leaving the release date field blank. The default option for the release date is = same as dataset release date, usually at this point its unknown. The option to provide a release date is important for the addition of files AFTER publication, so it can be seen that those files were not released with the original dataset.

**Comment by only1chunts** [comment]
Not fixed in Jesse's branch "fix/bugs" see image below for error message when I tried to add a file with no release date.
![image](https://cloud.githubusercontent.com/assets/6037145/15765791/45d9cef4-296b-11e6-98ba-179767282713.png)

If release date is not entered AND the dataset is already Public; then use Todays date, IF dataset is not published yet then leave date as NULL.

I've created another ticket (issue #36) to check if the file release date is updated when datasets are made public (only if date is NULL).
